### PR TITLE
Update example to EAPI=6, 2017

### PR DESCRIPTION
**Pull request by jonasstein**

### CODE REVIEW

File: ebuild-writing/eapi/text.xml [hunk 1]
```diff
@@ -42,11 +42,11 @@ Most developers prefer to set the EAPI version without quotes. However, the PMS
...
 
 <codesample lang="ebuild">
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # &#36;Id&#36;
 
-EAPI=5
+EAPI=6
 inherit eutils
```
Comment (gktrk) [inline ebuild-writing/eapi/text.xml]: I think we should also drop the inherit from there

**Comment by orlitzky** [comment]
I think the `inherit` was there to show that the EAPI line always comes first, but seeing an `EAPI=5` -> `EAPI=6` bump that retains `eutils.eclass` does make me twitch...

**Comment by gktrk** [comment]
I think it's a poor place to show that inherit comes after EAPI. We also hope to nuke this page altogether in the future. The PMS Appendix E contains almost the exact same thing.

**Comment by gktrk** [comment]
Anyway, merged in https://github.com/gentoo/devmanual.gentoo.org/compare/e098b2f...f4c6229.

I squashed the commits and mangled the commit message to prefix with the path of the directory. I also deleted the empty line right after `EAPI=6`

**Pull request closed without merging**